README.md

```diff
@@ -4,7 +4,9 @@
 [![CodeQL](https://github.com/cisagov/ansible-role-geoip2/workflows/CodeQL/badge.svg)](https://github.com/cisagov/ansible-role-geoip2/actions/workflows/codeql-analysis.yml)
 
 An Ansible role for installing a
-[MaxMind GeoIP2 database](https://www.maxmind.com/en/geoip2-databases).
+[MaxMind GeoIP2 database](https://www.maxmind.com/en/geoip2-databases). Additionally,
+it can install the [MaxMind `geoipupdate` tool](https://github.com/maxmind/geoipupdate)
+and add a SystemD service and timer to run the tool at regular intervals.
```

Comment:
Yep I was bamboozled somewhere along the way it seems. The [old website has a section specifically around the spelling of systemd](https://www.freedesktop.org/wiki/Software/systemd/#spelling):

> Yes, it is written systemd, not system D or System D, or even SystemD. And it isn't system d either. Why? Because it's a system daemon, and under Unix/Linux those are in lower case, and get suffixed with a lower case d. And since systemd manages the system, it's called systemd. It's that simple. But then again, if all that appears too simple to you, call it (but never spell it!) System Five Hundred since D is the roman numeral for 500 (this also clarifies the relation to System V, right?). The only situation where we find it OK to use an uppercase letter in the name (but don't like it either) is if you start a sentence with systemd. On high holidays you may also spell it sÿstëmd. But then again, [Système D](https://secure.wikimedia.org/wikipedia/en/wiki/System_D) is not an acceptable spelling and something completely different (though kinda fitting).